src/vendors/HTML5.jsx

```diff
@@ -3,12 +3,8 @@ import PropTypes from 'prop-types'
 import { findDOMNode } from 'react-dom'
 import vendorPropTypes from './vendor-prop-types'
 
-const AudioContext = window.AudioContext || window.webkitAudioContext
```

Comment:
I _think_ we only need to change this line. Something like:

```js
const AudioContext = typeof window === 'undefined' ? false : (window.AudioContext || window.webkitAudioContext)
```

Reply:
Awesome! If you get a chance and want to update this PR, I'll get it merged and released.